src/lib/helpers.ts

```diff
@@ -71,6 +71,19 @@ type SchemaParams = {
 };
 type SafeParseQueryStringParams<T> = GuardAndTransformParams<T> | SchemaParams;
 
+export enum GridDirection {
+  /** Right across the top row, then down to the next row, etc. */
+  RightDown = 'RightDown',
+  /** Down the leftmost column, then right to the next column, etc. */
+  DownRight = 'DownRight',
+  /** Right across the bottom row, then up to the next row, etc. */
+  RightUp = 'RightUp',
+  /** Up the leftmost column, then right to the next column, etc. */
+  UpRight = 'UpRight',
+  /** Down the rightmost column, then left to the next column, etc. - Rotate 180  */
```

Comment:
/** Left across the bottom row, then up to the next row, etc. */

Reply:
looks fine to me, thanks